app/assets/stylesheets/views/_completed-transaction.scss

```diff
@@ -1,3 +1,9 @@
+#organ-donor-registration-promotion {
```

Comment:
Can this selector live within the `.transaction-done` selector. That will make it clearer that the promotion will only ever appear on a completed transaction page.